src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const longTerm = 7;
+  const midTerm = 3;
+  const longDiscount = 50;
+  const midDiscount = 20;
+  const price = 40 * days;
```

Comment:
The calculation of the total rental cost (variable 'price') should be done after the condition checks. This is to ensure that we only perform the calculation if necessary, thus improving the performance of the function.